pokerbot.js

```diff
@@ -60,7 +62,11 @@ pokerbot.root = function (req, res, next) {
       }
     } else {
       responseText = util.getVotingResult(jiraId, pokerbot.pokerDataModel)
-      responseText = responseText + ' Thanks for voting.'
+      let unPlayedUsersName = util.getAllUnplayedUersForGame(pokerbot, jiraId)
```

Comment:
I think this variable should be called `unplayedUserNames` not `unPlayedUsersName`